hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -224,7 +224,7 @@ public void loadConfiguredFileSystem() throws Exception {
       this.fileSystemName = authorityParts[0];
 
       // Reset URL with configured filesystem
-      final String abfsUrl = this.getFileSystemName() + "@" + this.getAccountName();
+      final String abfsUrl = this.getFileSystemName() + "@" + authorityParts[1];
```

Comment:
Why is this change required ?

Reply:
This was buggy. authorityParts[1] is the actual FS to be loaded, getAccountName returns the FS that is created dynamically.